aws/terraform/README.md

```diff
@@ -0,0 +1,112 @@
+## Pre-requisites
+
+### Terraform
+
+[Download](https://www.terraform.io/downloads.html) and install terraform 0.14.x.
+
+### AWS
+
+AWS should be [configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html) with a profile. The `AWS_PROFILE` environment should be set to the profile that has been configured.
+
+The [awscli](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) can be used as follows to 
+confirm that it has been configured properly by running:
+
+```
+aws sts get-caller-identity
+```
+
+which should output your account information.
+
+## Setup
+
+### Infrastructure stack
+
+The infra sub-project provides some pre-requisite infrastructure for the metaflow service. For more details see the [README](aws/terraform/infra/README.md)
+
+Copy `example.tfvars` to `prod.tfvars` (or whatever environment name you prefer) and update that `env` name and the `region` as needed.
+
+Initialize the terraform:
+
+`cd infra && terraform init`
+
+Apply it:
+
+```
+terraform apply --var-file prod.tfvars
+```
+
+### Metaflow stack
+
+The metaflow sub-project provisions the metadata API, step functions, and an AWS Batch queue. For more details see the 
+[README](aws/terraform/metaflow/README.md)
+
+Copy `example.tfvars` to `prod.tfvars` (or whatever environment name you prefer) and update that `env` name and the `region` as needed.
+Metadata API authentication: the endpoint is exposed to the public internet via API Gateway, but only accessible to the IPs that match `access_list_cidr_blocks` (default is none).
+Additionally:
+* There are variables which govern the three compute environments associated with the AWS Batch queue that can be adjusted based on needs. 
+* The `enable_step_functions` flag can be set to false to not provision the step functions infrastructure. 
+* The `access_list_cidr_blocks` should be set to the network cidr blocks that will be accessing the metadata API.
+
+Initialize the terraform:
+
+`cd infra && terraform init`
+
+Apply it:
+
+```
+terraform apply --var-file prod.tfvars
+```
+
+Once the terraform executes, grab the output of `metaflow_profile_json` and put it in `~/.metaflowconfig/config_prod.json`
```

Comment:
A better approach would be for the user to store the JSON somewhere on their machine (`/path/to/json`) and invoke `metaflow configure import /path/to/json`

Reply:
I believe you're correct

@kldavis4 I recall you commenting on a new way to do this with a newer version of Terraform. Can you remind me?